plugins/versionpress/setup-hooks.php

```diff
@@ -659,6 +659,11 @@ function vp_admin_post_confirm_deactivation()
         FileSystem::remove(VERSIONPRESS_ACTIVATION_FILE);
     }
 
+    $wpConfigPath = \VersionPress\Utils\WordPressMissingFunctions::getWpConfigPath();
```

Comment:
This should be in the uninstallation script. We don't want to remove the constants when a user only deactivates VersionPress and activates it again (for example during an update).